homeassistant/components/axis/config_flow.py

```diff
@@ -191,13 +191,20 @@ def __init__(self):
             load_json, self.hass.config.path(CONFIG_FILE)
         )
 
+        # pylint: disable=no-member # https://github.com/PyCQA/pylint/issues/3167
+        self.context["title_placeholders"] = {
+            "name": discovery_info["hostname"],
+            "host": discovery_info[CONF_HOST],
```

Comment:
What's the difference between hostname and CONF_HOST, and why would we include both ?

Reply:
Host name is a configurable string that can help identify what device it is (device.local.) if not the host address will guide in what device it is

Reply:
I see. Can we drop `local` ?

Reply:
Absolutely